### ISSUE TITLE
Make a routine for reading tck files directly into LiFE

### ISSUE THREAD

**Issue by arokem** [francopestilli/life#48]
This currently requires a bit of a loop. First you need to convert the `tck` file into a `pdb` file using `mrtrix_tck2pdb`, then you need to use `mtrImportFibers` to read that one, and finally you need to save that into a mat file, which LiFE can read (e.g. here: https://github.com/francopestilli/life/blob/master/scripts/life_demo.m#L73). It would be nice to have one routine do all of that automatically.

**Comment by francopestilli** [comment]
Yep. I will work on this. I believe we have a little issue of coordinate frames with TCK files. Those are generally sorted in IMAGE space. With 0,0,0 Top-left-up. We store pd. and mat in ACPC. With 0,0,0, in the middle of the matrix. It owed be good to do some testing.
